group-generators/helpers/data-providers/degenscore/index.ts

```diff
@@ -0,0 +1,75 @@
+import axios from "axios";
+// eslint-disable-next-line no-restricted-imports
+import { ethers } from "ethers";
+// eslint-disable-next-line import/no-unresolved, @typescript-eslint/no-unused-vars
+import { BeaconResponse } from "./types";
+import { FetchedData } from "topics/group";
+
+export class DegenScoreProvider {
+  url: string;
+  headers: {
+    authorization: string;
+    accept: string;
+  };
+
+  provider: ethers.providers.JsonRpcProvider;
+
+  constructor() {
+    this.provider = new ethers.providers.JsonRpcProvider(
+      "https://eth.llamarpc.com"
+    );
+  }
+
+  public async getBeaconOwnersWithScore(_score: number) {
```

Comment:
You will have the same problem here with the Factory than with the getGitPoapHoldersByEventId function of the gitpoap provider

Reply:
should be working now.